### PR TITLE
Legacy (Vulcan Speed)

### PR DESCRIPTION
**Pull request by Razzy52**
Adds Vulcan Speed, bypasses well without flagging.

### CODE REVIEW

**Comment by Razzy52** [comment]
Doesn't change velocity stuff, not sure why that's included.

**Comment by CzechHek** [comment]
Why are there 2 VulcanYPort classes? Remove the empty method overrides, since those methods are no longer abstract.

**Comment by Razzy52** [comment]
made this sorta late at night, removed now. Hopefully that is right?

File: src/main/java/net/ccbluex/liquidbounce/features/module/modules/movement/speedmodes/other/VulcanYPort.kt [hunk 1]
```diff
@@ -47,71 +47,10 @@ class VulcanYPort : SpeedMode("VulcanYPort") {
         }
     }
 
-    override fun onMotion() {}
-
-    override fun onMotion(event: MotionEvent) {}
-
     override fun onDisable() {
         val scaffoldModule = Client.moduleManager.getModule(Scaffold::class.java)
```
Comment (CzechHek) [inline src/main/java/net/ccbluex/liquidbounce/features/module/modules/movement/speedmodes/other/VulcanYPort.kt:51]: wtf, client?

modules are now objects as well -> Scaffold.state

Comment (Razzy52) [reply src/main/java/net/ccbluex/liquidbounce/features/module/modules/movement/speedmodes/other/VulcanYPort.kt:51]: idfk what im doing so

Comment (NCPUser) [reply src/main/java/net/ccbluex/liquidbounce/features/module/modules/movement/speedmodes/other/VulcanYPort.kt:51]: fdp

Comment (Razzy52) [reply src/main/java/net/ccbluex/liquidbounce/features/module/modules/movement/speedmodes/other/VulcanYPort.kt:51]: what a guess

**Pull request closed without merging**